<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaixaFerramentaWidget</class>
 <widget class="QWidget" name="CaixaFerramentaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CaixaFerramentaWidget</string>
  </property>
  <widget class="QToolButton" name="selecao_tb">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Seleção / Edição de Objetos</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/novo.png</normaloff>:/icones/icones/novo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="objetos_tab_tb">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Objetos de Tabela</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/depsrefs.png</normaloff>:/icones/icones/depsrefs.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="caixatexto_tb">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Adicionar Caixa de Texto</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/textbox.png</normaloff>:/icones/icones/textbox.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="relacionamentos_tb">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Relacionamentos</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/convrelnn.png</normaloff>:/icones/icones/convrelnn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="objetos_bd_tb">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Objetos de Banco de Dados</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/editar.png</normaloff>:/icones/icones/editar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_selecionar_objeto">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/selmovobjeto.png</normaloff>:/icones/icones/selmovobjeto.png</iconset>
   </property>
   <property name="text">
    <string>Selecionar / Mover Objeto</string>
   </property>
  </action>
  <action name="action_adicionar_pontorel">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/inspontorelac.png</normaloff>:/icones/icones/inspontorelac.png</iconset>
   </property>
   <property name="text">
    <string>Adicionar Ponto</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Ponto ao Relacionamento</string>
   </property>
  </action>
  <action name="action_remover_pontorel">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/delpontorelac.png</normaloff>:/icones/icones/delpontorelac.png</iconset>
   </property>
   <property name="text">
    <string>Remover Ponto</string>
   </property>
   <property name="toolTip">
    <string>Remover Ponto do Relacionamento</string>
   </property>
  </action>
  <action name="action_funcagregada">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/aggregate.png</normaloff>:/icones/icones/aggregate.png</iconset>
   </property>
   <property name="text">
    <string>Função Agregada</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Função Agregada</string>
   </property>
  </action>
  <action name="action_conversao_tipos">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/cast.png</normaloff>:/icones/icones/cast.png</iconset>
   </property>
   <property name="text">
    <string>Conversão de Tipos</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Conversão de Tipos</string>
   </property>
  </action>
  <action name="action_conversao_codificacao">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/conversion.png</normaloff>:/icones/icones/conversion.png</iconset>
   </property>
   <property name="text">
    <string>Conversão de Codificação</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Conversão de Codificação</string>
   </property>
  </action>
  <action name="action_dominio">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/domain.png</normaloff>:/icones/icones/domain.png</iconset>
   </property>
   <property name="text">
    <string>Domínio</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Domínio</string>
   </property>
  </action>
  <action name="action_funcao">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/function.png</normaloff>:/icones/icones/function.png</iconset>
   </property>
   <property name="text">
    <string>Função</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Função</string>
   </property>
  </action>
  <action name="action_linguagem">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/language.png</normaloff>:/icones/icones/language.png</iconset>
   </property>
   <property name="text">
    <string>Linguagem</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Linguagem</string>
   </property>
  </action>
  <action name="action_classe_operadores">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/opclass.png</normaloff>:/icones/icones/opclass.png</iconset>
   </property>
   <property name="text">
    <string>Classe de Operadores</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Classe de Operadores</string>
   </property>
  </action>
  <action name="action_operador">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/operator.png</normaloff>:/icones/icones/operator.png</iconset>
   </property>
   <property name="text">
    <string>Operador</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Operador</string>
   </property>
  </action>
  <action name="action_familia_operadores">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/opfamily.png</normaloff>:/icones/icones/opfamily.png</iconset>
   </property>
   <property name="text">
    <string>Família de Operadores</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Família de Operadores</string>
   </property>
  </action>
  <action name="action_papel">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/role.png</normaloff>:/icones/icones/role.png</iconset>
   </property>
   <property name="text">
    <string>Papel</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Papel</string>
   </property>
  </action>
  <action name="action_esquema">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/schema.png</normaloff>:/icones/icones/schema.png</iconset>
   </property>
   <property name="text">
    <string>Esquema</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Esquema</string>
   </property>
  </action>
  <action name="action_sequencia">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/sequence.png</normaloff>:/icones/icones/sequence.png</iconset>
   </property>
   <property name="text">
    <string>Sequência</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Sequência</string>
   </property>
  </action>
  <action name="action_tabela">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/table.png</normaloff>:/icones/icones/table.png</iconset>
   </property>
   <property name="text">
    <string>Tabela</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Tabela</string>
   </property>
  </action>
  <action name="action_espaco_tabela">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/tablespace.png</normaloff>:/icones/icones/tablespace.png</iconset>
   </property>
   <property name="text">
    <string>Espaço de Tabela</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Espaço de Tabela</string>
   </property>
  </action>
  <action name="action_tipo">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/usertype.png</normaloff>:/icones/icones/usertype.png</iconset>
   </property>
   <property name="text">
    <string>Tipo</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Tipo</string>
   </property>
  </action>
  <action name="action_visao">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/view.png</normaloff>:/icones/icones/view.png</iconset>
   </property>
   <property name="text">
    <string>Visão</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Visão</string>
   </property>
  </action>
  <action name="action_coluna">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/column.png</normaloff>:/icones/icones/column.png</iconset>
   </property>
   <property name="text">
    <string>Coluna</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Coluna</string>
   </property>
  </action>
  <action name="action_restricao">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/constraint.png</normaloff>:/icones/icones/constraint.png</iconset>
   </property>
   <property name="text">
    <string>Restrição</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Restrição</string>
   </property>
  </action>
  <action name="action_regra">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/rule.png</normaloff>:/icones/icones/rule.png</iconset>
   </property>
   <property name="text">
    <string>Regra</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Regra</string>
   </property>
  </action>
  <action name="action_gatilho">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/trigger.png</normaloff>:/icones/icones/trigger.png</iconset>
   </property>
   <property name="text">
    <string>Gatilho</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Gatilho</string>
   </property>
  </action>
  <action name="action_gatilho_restricao">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/consttrigger.png</normaloff>:/icones/icones/consttrigger.png</iconset>
   </property>
   <property name="text">
    <string>Gatilho Restrição</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Gatilho Restrição</string>
   </property>
  </action>
  <action name="action_indice">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/index.png</normaloff>:/icones/icones/index.png</iconset>
   </property>
   <property name="text">
    <string>Índice</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Índice</string>
   </property>
  </action>
  <action name="action_relacionamento_1n">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/relationship1n.png</normaloff>:/icones/icones/relationship1n.png</iconset>
   </property>
   <property name="text">
    <string>Relacionamento 1:n</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Relacionamento 1:n</string>
   </property>
  </action>
  <action name="action_relacionamento_11">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/relationship11.png</normaloff>:/icones/icones/relationship11.png</iconset>
   </property>
   <property name="text">
    <string>Relacionamento 1:1</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Relacionamento 1:1</string>
   </property>
  </action>
  <action name="action_relacionamento_nn">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/relationshipnn.png</normaloff>:/icones/icones/relationshipnn.png</iconset>
   </property>
   <property name="text">
    <string>Relacionamento n:n</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Relacionamento n:n</string>
   </property>
  </action>
  <action name="action_relacionamento_gen">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/relationshipgen.png</normaloff>:/icones/icones/relationshipgen.png</iconset>
   </property>
   <property name="text">
    <string>Generalização</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Rel. de Generalização</string>
   </property>
  </action>
  <action name="action_relacionamento_dep">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/icones/icones/relationshipdep.png</normaloff>:/icones/icones/relationshipdep.png</iconset>
   </property>
   <property name="text">
    <string>Dependência</string>
   </property>
   <property name="iconText">
    <string>Dependência</string>
   </property>
   <property name="toolTip">
    <string>Adicionar Rel. de Dependência</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
